src/integrationTest/java/com/hivemq/helmcharts/LocalImageDeploymentIT.java

```diff
@@ -0,0 +1,39 @@
+package com.hivemq.helmcharts;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.k3s.K3sContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Testcontainers
+public class LocalImageDeploymentIT {
+
+    @Container
+    private final @NotNull K3sContainer container = new K3sContainer(DockerImageName
```

Comment:
Why do you use K3sContainer here instead of OperatorHelmChartContainer as in the other tests?
If there is a reason, maybe add a comment.

Reply:
Commented, is mostly to verify the functionality is working on the K3sContainer